ui/app/src/services/configuration.ts

```diff
@@ -95,6 +123,94 @@ export function getPreviewToolsConfig(site: string): Observable<PreviewToolsConf
   );
 }
 
+export function getAudiencesPanelConfig(site: string): Observable<ContentType> {
+  return getRawContent(site, `/targeting/targeting-config.xml`, 'studio').pipe(
+    map((content) => {
+      try {
+        return JSON.parse(content);
+      } catch (e) {
+        // Not JSON, assuming XML
+        let audiencesPanelContentType: ContentType = {
+          id: 'audiencesPanelConfig',
+          name: 'Audiences Panel Config',
+          type: 'unknown',
+          quickCreate: null,
+          quickCreatePath: null,
+          displayTemplate: null,
+          sections: null,
+          fields: null,
+          dataSources: null,
+          mergeStrategy: null
+        };
+
+        const xml = fromString(content);
+        const properties = Array.from(xml.querySelectorAll('property')).map((elem) => {
+
+          const name = getInnerHtml(elem.querySelector('name')),
+            label = getInnerHtml(elem.querySelector('label')),
+            description = getInnerHtml(elem.querySelector('description')),
+            type = getInnerHtml(elem.querySelector('type')),
+            defaultValue = getInnerHtml(elem.querySelector('default_value')),
+            hint = getInnerHtml(elem.querySelector('hint'));
+          let possibleValues: any[] | { value: string; }[];
+
+          if (elem.querySelectorAll('value').length > 0) {
+            possibleValues = Array.from(elem.querySelectorAll('value')).map((element) => {
+              const value = getInnerHtml(element);
+              return {
+                value: value
+              }
+            });
+          }else{
+            possibleValues = [];
+          }
+
+          return {
+            id: name,
+            name: label,
+            type: audienceTypesMap[type],
+            sortable: null,
+            validations: null,
+            defaultValue: defaultValue,
+            required: null,
+            fields: null,
+            values: possibleValues,
+            helpText: hint
+          }
+
+        });
+
+        audiencesPanelContentType.fields = createLookupTable(properties, 'id');
+
+        return audiencesPanelContentType;
+      }
+    })
+  );
+}
+
+export function getAudiencesPanelModel(): Observable<ContentInstance> {
```

Comment:
Rename to `fetchActiveProfile`

Add a todo to assess the location of this method. I'm not sure this belongs to this particular service.